### ISSUE TITLE
[Bug][Grid] The mobile version of the Pager in the Grid has no aria-label attribute

### ISSUE THREAD

**Issue by Yanko-Georgiev** [telerik/kendo-react#2845]
### Description

When the width of the Grid is less than 660px, the page number selector is rendered with a select that has no aria-label attribute.

### Steps To Reproduce

1. Open the following demo: https://stackblitz.com/edit/react-h5ayqzkn?file=app%2Fapp.tsx
2. Inspect the page number selection dropdown
3. Notice the aria-label attribute is not set

### Screenshots or video

<img width="1438" alt="Image" src="https://github.com/user-attachments/assets/ca9f4b9a-ac91-4648-8908-fd0fdaebde32" />

### Actual Behavior

In mobile view, the Pager inside the Grid has no aria-label

### Expected Behavior

The mobile version of the Pager in the Grid should have an aria-label

### Browser

Chrome

### Browser version

latest

### OS type

MacOS

**Comment by filipKovachev** [comment]
Still reproducible in version 14.2.0:

https://stackblitz.com/edit/react-l5bmbfmn?file=app%2Fapp.tsx

**Comment by Sasho32** [comment]
The fix is available in v14.4.0-develop.7.
